.github/workflows/publish_image.yml

```diff
@@ -140,3 +140,58 @@ jobs:
     with:
       version: ${{ needs.settings.outputs.image_version }}
     secrets: inherit
+
+  update-remote-repo:
+    needs:
+      - settings
+      - publish
+    runs-on: ubuntu-latest
+    env:
+      service_to_update: 'studio'
+      cli_repo_owner: supabase
+      cli_repo: cli
+      cli_repo_main_branch: 'main'
+      cli_repo_pr_title: 'New Studio Version'
+      github_user: 'Ivan Vasilov'
+      github_user_email: 'vasilov.ivan@gmail.com'
+
+    steps:
+      - name: Setup Git config
+        run: |
+          git config --global user.name '${{ env.github_user }}'
+          git config --global user.email '${{ env.github_user_email }}'
+
+      - name: Checkout remote repository
+        uses: actions/checkout@v2
+        with:
+          repository: '${{ env.cli_repo_owner }}/${{ env.cli_repo }}'
+          token: ${{ secrets.USER_ACCESS_TOKEN }}
+          path: '${{ env.cli_repo }}'
+
+      - name: Create a new branch
+        run: |
+          cd '${{ env.cli_repo }}'
+          git checkout -b '${{ env.service_to_update }}/${{ needs.settings.outputs.image_version }}'
+
+      - name: Update file
+        run: |
+          cd '${{ env.cli_repo }}'
+          sed -i 's/${{ env.cli_repo_owner }}\/${{ env.service_to_update }}:.*/${{ env.cli_repo_owner }}\/${{ env.service_to_update }}:${{ needs.settings.outputs.image_version }}"/' internal/utils/misc.go
+
+      - name: Commit changes
```

Comment:
You could also combine these three steps into a single simpler one using a GH action

Example

```
- name: Create Pull Request
        uses: peter-evans/create-pull-request@v4.2.0
        with:
          add-paths: package*.json
          branch: studio-${{ github.event.client_payload.version }}
          body: Bump the import package to version ${{ github.event.client_payload.version }}.
          base: main
          title: '${{ github.event.client_payload.message }}'
          reviewers: ...
          committer: ...
          author: ...
```

Reply:
I've looked into this action, it looks like it only supports PRs towards its own repo. Even if it supports other repos, the job still has to check the repo, make changes and then use this action to commit and push (which is easiest part of the job).

Reply:
The idea would be that supabase only emits an event and the cli repo would then do the changes (as the cli repo knows it's structure) and create a PR on it's own repo.

Trigger would be the dispatched event (as outlined in the other comment)

Reply:
but for the sake of not refactoring the action, no hard feelings, can leave it as-is